modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -2936,6 +2936,20 @@ definition {
 		Workflow.openWorkflowListView();
 
 		Workflow.viewDefinitionUnpublished(workflowDefinitionTitle = "New Workflow Definition Title");
+
+		Workflow.gotoEditWorkflowDefinition(workflowDefinitionTitle = "New Workflow Definition Title");
+
+		ProcessBuilderKaleoDesignerReact.addConnector(
+			connectorIn = "end",
+			connectorLabel = "Start Task Connector",
+			connectorOut = "start",
+			key_direction = "top");
+
+		PortletEntry.publish();
+
+		Navigator.gotoBack();
+
+		Workflow.viewDefinitionPublished(workflowDefinitionTitle = "New Workflow Definition Title");
```

Comment:
@davisaints I am reading the title and description of the test and wondering if we actually need to publish it if the main intent of this test case is to make sure a workflow definition is saved as unpublished. Same with the other test, maybe we can use the term Not Published instead of Draft to make it consistent with the UI.

Reply:
It makes sense! I will remove the publish step and change the test name and description